pkg/ingress/apisix_cluster_config.go

```diff
@@ -62,7 +64,6 @@ func (c *apisixClusterConfigController) run(ctx context.Context) {
 		go c.runWorker(ctx)
 	}
 	<-ctx.Done()
-	c.workqueue.ShutDown()
```

Comment:
It's not related to PR change. If you wanna change them to use `defer`, just in another PR.

Reply:
If not change them. Wiil cause goroutine leak in this pr. When re-create *controller.

Reply:
Did you mean the case that cache sync failure?